tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -45,7 +46,7 @@ class ExpectSuite extends FunSuite with DiffAssertions {
         import MetacIndexExpect._
         assertNoDiff(loadObtained, loadExpected)
       }
-      test("metac-metacp.expect.diff") {
+      test("metac-metacp.diff") {
```

Comment:
Why change the test name?

Reply:
To play nicer with scalatest -z option that allows you to filter tests by regexp. I wasn't able to run metacp.expect standalone before.

Reply:
The test is called like that after the file it's loading. Let's rename that file, then.

Reply:
Done.